<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableWidget" name="tableWidgetGoods">
            <column>
             <property name="text">
              <string>商品名称</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>编号</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>总数量</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>进价</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>售价</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>生产厂家</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>生产日期</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>保质期</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>操作1</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>操作2</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>操作3</string>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>备注</string>
             </property>
             <property name="font">
              <font>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Policy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>80</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Policy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonAddGoods">
              <property name="text">
               <string>添加商品</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Policy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonExit">
              <property name="text">
               <string>退出</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Policy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Policy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>80</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTableWidget" name="tableWidgetSales">
            <column>
             <property name="text">
              <string>订单号</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>数量</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>折扣率</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>订单时间</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>联系电话</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>操作1</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>操作2</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonSalesAdd">
              <property name="text">
               <string>增加销售数据</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Policy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonExit2">
              <property name="text">
               <string>返回</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QTabWidget" name="tabWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>771</width>
          <height>481</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="goodsInquiry">
         <attribute name="title">
          <string>商品信息查询</string>
         </attribute>
         <widget class="QTableWidget" name="tableWidgetGoodsInquiry">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>711</width>
            <height>391</height>
           </rect>
          </property>
          <column>
           <property name="text">
            <string>商品名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>编号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>总数量</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>进价</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>售价</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>生产厂家</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>生产日期</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>保质期</string>
           </property>
          </column>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>371</width>
            <height>26</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>查询关键字：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>商品名称</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>商品编号</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>生产厂家</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonInquiry1">
             <property name="text">
              <string>查询</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="goodsInquiryAdvanced">
         <attribute name="title">
          <string>商品信息高级检索</string>
         </attribute>
         <widget class="QTableWidget" name="tableWidgetGoodsInquiryAdvanced">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>711</width>
            <height>391</height>
           </rect>
          </property>
          <column>
           <property name="text">
            <string>商品名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>编号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>总数量</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>进价</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>售价</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>生产厂家</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>生产日期</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>保质期</string>
           </property>
          </column>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>214</width>
            <height>26</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLineEdit" name="lineEdit_3"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonInquiry1Advanced">
             <property name="text">
              <string>高级检索</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="salesInquiry">
         <attribute name="title">
          <string>销售信息查询</string>
         </attribute>
         <widget class="QTableWidget" name="tableWidgetSalesInquiry">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>711</width>
            <height>391</height>
           </rect>
          </property>
          <column>
           <property name="text">
            <string>订单号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>数量</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>折扣率</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>订单日期</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>联系电话</string>
           </property>
          </column>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>21</x>
            <y>10</y>
            <width>371</width>
            <height>26</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>查询关键字：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_2">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>21</height>
              </size>
             </property>
             <property name="maxVisibleItems">
              <number>3</number>
             </property>
             <item>
              <property name="text">
               <string>编号</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>联系电话</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonInquirt2">
             <property name="text">
              <string>查询</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="salesInquiryAdvanced">
         <attribute name="title">
          <string>销售信息高级检索</string>
         </attribute>
         <widget class="QTableWidget" name="tableWidgetSalesInquiryAdvanced">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>711</width>
            <height>391</height>
           </rect>
          </property>
          <column>
           <property name="text">
            <string>订单号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>数量</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>折扣率</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>订单日期</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>联系电话</string>
           </property>
          </column>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>214</width>
            <height>26</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLineEdit" name="lineEdit_4"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonInquiry2Advanced">
             <property name="text">
              <string>高级检索</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButtonBackToMain">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>490</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件管理</string>
    </property>
    <addaction name="actionopenFile"/>
    <addaction name="separator"/>
    <addaction name="actionsaveFile"/>
    <addaction name="separator"/>
    <addaction name="actionsaveFileAs"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>辅助功能</string>
    </property>
    <addaction name="actioninquiry"/>
    <addaction name="separator"/>
    <addaction name="actiondataAnalysis"/>
    <addaction name="separator"/>
    <addaction name="actionnonStock"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopenFile">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actionsaveFile">
   <property name="text">
    <string>保存文件</string>
   </property>
  </action>
  <action name="actionsaveFileAs">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actioninquiry">
   <property name="text">
    <string>信息查询</string>
   </property>
  </action>
  <action name="actiondataAnalysis">
   <property name="text">
    <string>数据分析</string>
   </property>
  </action>
  <action name="actionnonStock">
   <property name="text">
    <string>缺货管理</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
